build.py

```diff
@@ -336,7 +336,8 @@ def tag(self):
             logging.info("Clang 7 will produce the alias Clang 7.0")
             subprocess.check_call("docker tag %s %s" % (self.created_image_name,
             self.tagged_image_name.replace("clang7", "clang70")), shell=True)
-
+            subprocess.check_call("docker tag %s/clang7 %s/clang70" %
+            (self.variables.docker_username, self.variables.docker_username), shell=True)
```

Comment:
I don't understand. Isn't this the exact same command as executing a few lines earlier?

Reply:
the previous command contains the tag based on Conan version (e,g, 1.19.2). This new command creates a tag to the latest version as there is no tag referred.